templates/admin/plugins/reactions.tpl

```diff
@@ -26,6 +26,8 @@
 
 
 
-<button id="save" class="floating-button mdl-button mdl-js-button mdl-button--fab mdl-js-ripple-effect mdl-button--colored">
-    <i class="material-icons">save</i>
-</button>
+<div class="floating-button">
+	<button id="save" class="btn btn-primary position-fixed bottom-0 end-0 px-3 py-2 mb-4 me-4 rounded-circle fs-4" type="button" style="width: 64px; height: 64px;">
+		<i class="fa fa-fw fas fa-floppy-disk"></i>
+	</button>
+</div>
```

Comment:
can use `<!-- IMPORT admin/partials/save_button.tpl -->` to use the core save button

Reply:
done